common/xbps-src/shutils/common.sh

```diff
@@ -140,8 +140,8 @@ msg_warn_nochroot() {
 
 msg_normal() {
     if [ -z "$XBPS_QUIET" ]; then
-        # normal messages in bold
-        [ -n "$NOCOLORS" ] || printf "\033[1m"
+        # normal messages in bright bold white
+        [ -n "$NOCOLORS" ] || printf "\033[97;1m"
```

Comment:
Seeing the code for msg_{red,warn}, I suspect that this will need to be changed to `\033[97m\033[1m`